Gems/ROS2/Code/Include/ROS2/Manipulation/MotorizedJoints/JointMotorControllerComponent.h

```diff
@@ -13,6 +13,7 @@
 #include <AzCore/Serialization/SerializeContext.h>
 #include <ImGuiBus.h>
 #include <ROS2/Manipulation/MotorizedJoints/JointMotorControllerConfiguration.h>
+#include <ROS2/ROS2Bus.h>
```

Comment:
No need to include _ROS2Bus_ here:
```suggestion
#include <builtin_interfaces/msg/time.hpp>
```